nodejs/demos/trivial/adams.js

```diff
@@ -119,17 +97,18 @@ Adams.prototype.beginTransaction = function() {
  * Lookup for the Trivia entity.
  */
 Adams.prototype.lookup = function() {
-  // Get entities by key.
-  this.datastore.lookup({
-    readOptions: {
-      // Set the transaction, so we get a consistent snapshot of the
-      // value at the time the transaction started.
-      transaction: this.transaction
-    },
-    // Add one entity key to the lookup request, with only one
-    // `path` element (i.e. no parent).
-    keys: [{ path: [{ kind: 'Trivia', name: 'hgtg' }] }]
-  }).execute((function(err, result) {
+  this.datastore = googleapis.datastore({
```

Comment:
Should create the datastore instance outside of the lookup function so its not re-created every time (would be inefficient)